public_html/wp-content/plugins/wordcamp-speaker-feedback/views/view-feedback.php

```diff
@@ -41,6 +44,69 @@
 <div class="speaker-feedback">
 	<h2><?php esc_html_e( 'Session Feedback', 'wordcamporg' ); ?></h2>
 
+	<p class="speaker-feedback__notice">
+		<?php if ( $is_session_speaker ) : ?>
+			<?php
+			esc_html_e( '
+				Email notifications are sent up to once per day when new feedback submissions have been approved.
+				Disabling this will turn off feedback notifications on all WordCamp sites.
+				',
+				'wordcamporg'
+			);
+			?>
+		<?php else : ?>
+			<?php
+			esc_html_e( '
+				Email notifications to speakers are sent up to once per day when new feedback submissions have been
+				approved. Disabling these emails for a speaker will turn off feedback notifications for them on all
+				WordCamp sites.
+				',
+				'wordcamporg'
+			);
+			?>
+		<?php endif; ?>
+	</p>
+
+	<?php foreach ( $session_speakers as $session_speaker ) :
+		if ( $is_session_speaker && get_current_user_id() !== $session_speaker ) {
+			continue;
+		}
+		$user = new WP_User( $session_speaker );
+		$notifications_opt_out = get_user_meta( $user->ID, SPEAKER_OPT_OUT_KEY, true );
+		?>
+		<div class="speaker-feedback__notifications <?php echo ( $notifications_opt_out ) ? 'is-disabled' : ''; ?>">
+			<span id="sft-notifications-description">
```

Comment:
This isn't unique in the organizer's view if there's more than one speaker. appending the speaker ID should be good to make it unique.

Reply:
Thanks, good catch. 0b4d5a90